sdk/vision/azure-ai-vision-imageanalysis/src/samples/java/com/azure/ai/vision/imageanalysis/pom.xml

```diff
@@ -12,17 +12,15 @@
       <artifactId>azure-ai-vision-imageanalysis</artifactId>
       <version>1.0.0-beta.1</version>
     </dependency>
-    <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-api -->
+    <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-nop -->
```

Comment:
Tagging @sandeep-sen to get your thoughts on having a pom.xml in `src/samples` directory.

Reply:
We started an IM chat with @srnagar and @sandeep-sen to discuss this. Will make changes in another PR if needed.